server/game/cards/01_SOR/units/MaceWinduPartyCrasher.ts

```diff
@@ -14,6 +14,7 @@ export default class MaceWinduPartyCrasher extends NonLeaderUnitCard {
             title: 'Ready Mace Windu',
             when: {
                 onCardDefeated: (event, context) =>
+                    // TODO: update trigger condition so that defender being defeated by attacker at any point during the attack will work
```

Comment:
Sorry I'm realizing I wrote this in an ambiguous way - do you mind updating instances of this to instead say `... being defeated by attacker at the 'on attack' stage will also work`. That way we'll know clearly what the issue is when we come back around to it.